Documentation/CallGuide.md

```diff
@@ -113,13 +113,17 @@ peerConnection.addEventListener( 'signalingstatechange', event => {
 	};
 } );
 
-peerConnection.addEventListener( 'addstream', event => {
+peerConnection.addEventListener( 'track', event => {
 	// Grab the remote stream from the connected participant.
-	remoteMediaStream = event.stream;
+	const connection = event.target;
+	const stream = connection._remoteStreams.values().next().value;
```

Comment:
Don't. That's private and could go away any time. The stream is part of the event data. But note there could be streamless tracks.

Reply:
Ok. Then I'll do using tracks :)